data/data/gcp/bootstrap/main.tf

```diff
@@ -13,6 +13,9 @@ resource "google_compute_instance" "bootstrap" {
 
   network_interface {
     subnetwork = var.subnet
+
+    access_config {
```

Comment:
how do this enable public ip access.

Reply:
See the description of `access_config` here: https://www.terraform.io/docs/providers/google/r/compute_instance.html

Reply:
It can be left empty to get a random ip assigned, or you can create a public address resource and assign it to the instance here.

Reply:
can we include this detail in the commit message.

Reply:
Done

Reply:
> It can be left empty to get a random ip assigned, or you can create a public address resource and assign it to the instance here.

I will create the public address resource for [CORS-1123]( https://jira.coreos.com/browse/CORS-1123)